desktop/src/custom_event.rs

```diff
@@ -19,6 +19,9 @@ pub enum RuffleEvent {
     /// The user requested to close the current SWF.
     CloseFile,
 
+    /// The user requrested to enter full screen.
+    EnterFullscreen,
+
     /// The user requested to exit full screen.
     ExitFullScreen,
```

Comment:
There is an inconsistency here: `EnterFullscreen` vs `ExitFullScreen`.

Reply:
Fixed.